src/tests/flyer.test.ts

```diff
@@ -276,6 +276,45 @@ describe('getFlyersByCategorySlug tests', () => {
   });
 });
 
+describe('getAllFlyerCategories tests', () => {
+  test('query 3 different flyers with 2 different categories', async () => {
```

Comment:
Our goal should be to have really descriptive test names -- I know a lot of the other tests in this codebase are structured like the name you've written here, but I'd suggest changing it up to reflect what we're actually testing. Our goal here is to show that our query filters out duplicate slugs, so something like `Handle non-unique CategorySlugs across flyers in collection` or `ensure only unique categorySlugs are returned` would make more sense here